public/js/user.js

```diff
@@ -406,10 +437,12 @@ function contentInfowindow(site) {
 
 function contentEditwindow(site) {
   var html = "<div class='site-bubble bubble-form'>";
-  html += "<form method='post' id='"+ site.id +"'>";
+  html += "<form method='post' id='"+ site.id +"' enctype='multipart/form-data'>";
   html += "<div class='form-group'><input type='text' required='true' placeholder='Site Name' title='Site Name' class='form-control input-sm' value='"+ site.name + "' id='name' name='name'></div>";
   html += "<div class='form-group'><input type='url' class='form-control input-sm' placeholder='Site URL' title='Site URL' value='"+ site.url + "' name='url' id='url'></div>";
-  html += "<div class='form-group'><input type='url' class='form-control input-sm' placeholder='Image' title='Image' value='"+ site.image + "' name='image' id='image'></div>";
+  html += "<div class='form-group'><p>Image: </p><input type='file' title='Image' name='image' id='image' onchange='readImage(event)'></div>";
+  html += "<div id='image-hint'>square (prefer 80x80) jpeg or png up to " + Math.round(maxImageUploadSize/1024) + " kb</div>"
```

Comment:
Thanks for including this tip; however, I'd suggest tweaking the styling a little. Right now, it looks like this:
<img width="307" height="343" src="https://user-images.githubusercontent.com/860834/61191742-dd918300-a67b-11e9-8c20-1de490cb40ef.png" />
where the tip can be lost or feel like it belong with the contact name. I'd suggest styling it closer to something like this:
<img width="299" height="337" src="https://user-images.githubusercontent.com/860834/61191766-0154c900-a67c-11e9-8325-70502b2e9133.png" />
that is, put it between the "Image:" label and the file selector.